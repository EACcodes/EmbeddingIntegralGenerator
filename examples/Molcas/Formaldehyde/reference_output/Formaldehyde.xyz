4
Bohr
 C   6.792095      11.751729      16.079757 
 O   5.511199      12.136307      14.239049 
 H   6.668460       9.956442      17.197795 
 H   8.196404      13.161803      16.802606 
