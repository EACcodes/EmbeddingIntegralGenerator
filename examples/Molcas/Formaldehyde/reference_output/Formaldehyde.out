
********************************************************************************
********************************************************************************
**********                                                            **********
                        EMBEDDING INTEGRAL GENERATOR                            
**********                                                            **********
********************************************************************************
********************************************************************************

********************************************************************************
**********                                                            **********
                        CONFIGURATION OF THIS CALCULATION                       

Keyword                                 Value
-------                                 -----
BASIS FILE                              basis_set_TZ
CARTESIAN                               0
EMBEDDING INTS REAL                     0
EMBEDDING POTENTIAL FILE                embpot.dat
GRID DIMENSIONS                         100  80  120  
LATTICE VECTORS                         14.3065  0  0  
                                        -7.15324  12.3898  0  
                                        0  0  17.0087  
NUM THREADS                             16
OUTPUT FORMAT                           MOLCAS
SHIFT VECTOR                            Not defined, use default:
                                        0  0  0  
XYZ FILE                                Formaldehyde.xyz
**********                                                            **********
********************************************************************************

Running with OpenMP using 16 threads
***********************
*                      
* Embedding Integrals  
*                      
***********************

  Read grid data and embedding potential wall time(s): 0.449 cpu time(s): 0.38

* Reciprocal space integration scheme
  Calculate embedding integrals, real space part wall time(s): 28.543 cpu time(s): 209.48
  Calculate embedding integrals, reciprocal space part wall time(s): 34.037 cpu time(s): 254.1

  Calculate embedding integrals, sorting and printing wall time(s): 34.209 cpu time(s): 256.63
TOTAL EXECUTION TIME  wall time(s): 34.699 cpu time(s): 257.02
